Makefile

```diff
@@ -93,7 +93,9 @@ packaging/debian/changelog: packaging/debian/changelog.in
 
 $(WEBPACK_TEST): $(COCKPIT_REPO_STAMP) $(shell find src/ -type f) package.json webpack.config.js
 	$(MAKE) package-lock.json && NODE_ENV=$(NODE_ENV) node_modules/.bin/webpack
-	[ -f dist/index.js.LICENSE.txt.gz ] || gzip </dev/null >dist/index.js.LICENSE.txt.gz
+	if [ "$$NODE_ENV" = "development" ]; then \
+		[ -f dist/index.js.LICENSE.txt.gz ] || gzip </dev/null >dist/index.js.LICENSE.txt.gz; \
```

Comment:
As you said, we don't need the `-f` test here, please drop.

Reply:
Yup, this was a test commit. Dropped `-f`.